src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/dtos/ReportedAdjudicationDto.kt

```diff
@@ -206,7 +211,33 @@ data class CombinedOutcomeDto(
 @Schema(description = "Previous DIS1/2 issues")
 data class DisIssueHistoryDto(
   @Schema(description = "Previous issuing officer")
-  var issuingOfficer: String,
+  val issuingOfficer: String,
   @Schema(description = "Previous date time of form issued")
-  var dateTimeOfIssue: LocalDateTime,
+  val dateTimeOfIssue: LocalDateTime,
+)
+
+@Schema(description = "punishment")
+data class PunishmentDto(
+  @Schema(description = "punishment type")
+  val type: PunishmentType,
+  @Schema(description = "optional privilege type")
+  val privilegeType: PrivilegeType? = null,
+  @Schema(description = "optional other privilege type")
+  val otherPrivilege: String? = null,
+  @Schema(description = "optional stoppage of earnings percentage")
+  val stoppagePercentage: Int? = null,
+  @Schema(description = "latest punishment schedule")
+  val schedule: PunishmentScheduleDto,
+)
+
+@Schema(description = "punishment schedule")
+data class PunishmentScheduleDto(
+  @Schema(description = "days punishment will last")
+  val days: Int,
+  @Schema(description = "optional start date of punishment")
+  val startDate: LocalDate? = null,
+  @Schema(description = "optional end date of punishment")
+  val endDate: LocalDate? = null,
+  @Schema(description = "optional punishment suspended until data")
```

Comment:
Minor one - should this be
```suggestion
  @Schema(description = "optional punishment suspended until date")
```